sys/include/net/gnrc/rpl.h

```diff
@@ -9,8 +9,9 @@
  */
 
 /**
- * @defgroup    net_gnrc_rpl  New RPL
- * @ingroup     net
+ * @defgroup    net_gnrc_rpl  RPL
+ * @ingroup     net_gnrc
+ * @brief       RPL implementation for GNRC
```

Comment:
> Helper functions to me are auxiliary functions that can easily replaced by an external max. 5-line code

O.K. I was more focused on the functions *defined* in the header file. At least I see no "right or wrong" here and also it's not affecting functionality.
Thanks for clarifying.